python/skupper_router/management/skrouter.json

```diff
@@ -1254,6 +1254,13 @@
                     "type": ["up", "down"],
                     "description": "The operational status of TCP socket listener: up - the service is active and incoming connections are permitted; down - the service is not active and incoming connection attempts will be refused.",
                     "create": false
+                },
+                "idleTimeoutSeconds": {
+                    "type": "integer",
+                    "default": 300,
+                    "description": "The idle timeout, in seconds, for connections through this listener.  If no data is transferred over the connection for this time interval, the connection shall be closed. A value of zero disables the timeout",
```

Comment:
If users want to turn this feature off by setting 0 for idleTimeoutSeconds, there will have to be a skupper command line command that will need to be added to allow this to be done. Just like how @grs added a command line option to set dataConnectionCount

Reply:
I'll add the control-plane devs to the review as well as reach out to them directly.

Reply:
(issue created in skupper project)

Reply:
* https://github.com/skupperproject/skupper/issues/1167

lets link it both ways and from all places :)